src/Http/Requests/StoreLargoSession.php

```diff
@@ -112,13 +112,13 @@ protected function getAffectedAnnotations($dismissed, $changed)
      */
     protected function imageAnotationsBelongToVolumes($annotations, $volumes)
     {
-        $chunkedI = array_chunk($annotations,config('biigle.db_param_limit')-1); 
```

Comment:
Why did you remove the `-1` everywhere?

Reply:
Using the volume ID count to set the chunk size resolved the error I had previously addressed with a value of -1. So there is no longer a need for it.